source/php/Component/Date/date.json

```diff
@@ -10,14 +10,23 @@
         "labelsPlural": []
     },
     "description": {
-        "date": "The date",
+        "timestamp": "The date",
         "time_since": "Will return the date since.",
         "time_since_cap": "How long back to spell out time since",
         "format": "Present date's in this format. Default: D d M Y",
         "action": "What to do with the inputted date: formatDate, timesince, timeuntil",
         "labels": "Array containing translations valid keys: year, month, week, day, hour, minute, second",
         "labelsPlural": "Array containing translations valid keys: year, month, week, day, hour, minute, second"
     },
+    "types": {
+        "timestamp": "string",
+        "time_since": "boolean",
+        "time_since_cap": "string",
+        "format": "string",
+        "action": "string",
+        "labels": "array",
+        "labelsPlural": "array"
```

Comment:
These should be object|array since we cant use keys in arrays for our default values in json